packages/shared/prisma/schema.prisma

```diff
@@ -753,15 +753,30 @@ model Model {
     inputPrice      Decimal?  @map("input_price")
     outputPrice     Decimal?  @map("output_price")
     totalPrice      Decimal?  @map("total_price")
-    unit            String // TOKENS, CHARACTERS, MILLISECONDS, SECONDS, REQUESTS, or IMAGES
+    unit            String? // TOKENS, CHARACTERS, MILLISECONDS, SECONDS, REQUESTS, or IMAGES
     tokenizerId     String?   @map("tokenizer_id")
     tokenizerConfig Json?     @map("tokenizer_config")
+    Price           Price[]
 
     @@unique([projectId, modelName, startDate, unit])
     @@index(modelName)
     @@map("models")
 }
 
+model Price {
+    id        String   @id @default(cuid())
+    createdAt DateTime @default(now()) @map("created_at")
+    updatedAt DateTime @default(now()) @updatedAt @map("updated_at")
+    modelId   String   @map("model_id") // Model is already linked to project (or default), so we don't need projectId here
+    Model     Model    @relation(fields: [modelId], references: [id], onDelete: Cascade)
+    itemName  String   @map("item_name")
```

Comment:
what does itemName mean? Maybe we can find better naming here.

Reply:
Agree, will come up with more options here

Reply:
1.	usageType - Generic and flexible, covers any type of usage measurement
	2.	metricType - Generic term for different types of measurements
	3.	countType - Simple and clear for counting different elements
	4.	measureType - Indicates what kind of measurement we're tracking
	5.	llmMetricType - Specific to LLM but generic about what's being measured
	6.	usageCategory - Categorizes different types of usage
	7.	consumptionType - Focuses on what's being consumed
	8.	processType - If you're tracking different stages of processing
	
	
	what do you like best?

Reply:
I like `usageType` - let's go with that